Quote and encoding error

Tests are failing, probably due to some update lib. One example:

FAILED tests/server/blueprints/clinvar/test_clinvar_views.py::test_clinvar_update_submission - TypeError: quote() doesn't support 'encoding' for bytes

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -17,6 +17,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Page crashing when a user tries to edit a comment that was removed
 - Warning instead of crashed page when attempting to retrieve a non-existent Phenopacket
 - Fixed StJude ProteinPaint gene link (URL change)
+- Freeze of werkzeug library to version<2.2.3 to avoid problems resulting from the consequential upgrade of the Flask lib
 
 
 ## [4.66]

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,5 +1,5 @@
 # Everything needed in production
-werkzeug
+werkzeug<2.2.3
 Flask>=2.0
 Flask-Bootstrap
 Flask-CORS

